CMSIS/Core/Include/core_cm7.h

```diff
@@ -876,8 +876,11 @@ typedef struct
 #define SCB_CACR_FORCEWT_Pos                2U                                            /*!< SCB CACR: FORCEWT Position */
 #define SCB_CACR_FORCEWT_Msk               (1UL << SCB_CACR_FORCEWT_Pos)                  /*!< SCB CACR: FORCEWT Mask */
 
-#define SCB_CACR_ECCEN_Pos                  1U                                            /*!< SCB CACR: ECCEN Position */
-#define SCB_CACR_ECCEN_Msk                 (1UL << SCB_CACR_ECCEN_Pos)                    /*!< SCB CACR: ECCEN Mask */
+#define SCB_CACR_ECCEN_Pos                  1U                                            /*!< \deprecated SCB CACR: ECCEN Position */
+#define SCB_CACR_ECCEN_Msk                 (1UL << SCB_CACR_ECCEN_Pos)                    /*!< \deprecated SCB CACR: ECCEN Mask */
+
+#define SCB_CACR_ECCDIS_Pos                 1U                                            /*!< SCB CACR: ECCEN Position */
```

Comment:
Copy-paste: Should read ECCDIS

Reply:
The new macros are ok but I do not agree in keeping old ones. ECCEN ones should be removed as their names are misleading. The bit has inverse logic, so a developer using ECCEN would imagine that a 1 means ECC enable, while it's ECC disable.

Reply:
Yes, you're right, the old names are misleading. But removing them would break existing projects if the old names are used. That's why we mark the old names as deprecated but leave them in for compatibility reasons.

Reply:
My personal view is that it's better to break existing projects and force them to align to correct use rather than creating confusion for new users of them or new projects with presence of both ECCDIS and ECCEN.